frontend/src/atoms/UserUsername.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export function UserUsername({ username }) {
```

Comment:
I'd use `UserUserName`.

There is already existing `UserName` component, so let's keep it consistent.

Reply:
lowercase `username` is ok